deployment/scripts/sonarqube.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "SonarQube Scan"
+echo "Directory: $PWD"
```

Comment:
What is `$PWD` expected to return at this point? Do we know the runner's current working directory on the filesystem when this script is called?

I ask because I'd be concerned that its value might change unexpectedly if the runner itself is ever called differently or if the runner ever calls this script differently than usual. Since we're `mkdir`ing a bunch of extra folders and files, that could put things in different places, and I don't know if that's okay. 

Should use something more predictable and static? How about something `/tmp/` or `$TMPDIR`? Or something like [this $SCRIPT_DIR one-liner](https://stackoverflow.com/a/246128)?

Reply:
That value seems to be /var/lib/jenkins/workspace/cloudigrade-cloudigrade-pr-check, so probably a good idea to keep  things as Jenkins probably clean things up afterward.